### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by hana-acciaioli**

### CODE REVIEW

File: src/components/Controls/Controls.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      return currentState + 1;
+    });
+  };
+  const pantsHandler = (event) => {
+    setPants(event.target.value);
+    setCountPants((currentState) => {
+      return currentState + 1;
+    });
+  };
+  // const catchphraseInputHandler = (event) => {
```
Comment (aidanliddiard) [inline src/components/Controls/Controls.js]: You can delete this to clean up your code

File: src/components/Controls/Controls.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      alert('please enter a valid catchphrase');
+    } else {
+      setCatchphrases((currentState) => [...currentState, inputValue]);
+      setInputValue('');
+    }
+  };
+
+  const handleKeyPress = (e) => {
+    if (e.keyCode === 13) {
+      addCatchphraseHandler();
```
Comment (aidanliddiard) [inline src/components/Controls/Controls.js:48]: Cool implementation!!

File: src/components/layout/Header/Header.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
```
Comment (aidanliddiard) [inline src/components/layout/Header/Header.js]: If you don't have a header you can delete this file/folder

File: src/components/layout/Footer/Footer.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
```
Comment (aidanliddiard) [inline src/components/layout/Footer/Footer.js]: If you don't have a footer you can delete this file/folder

**Review by aidanliddiard — COMMENTED** [review]
Looks super great! Love the styling!

File: src/components/Controls/Controls.js (hunk 2 above)
Comment (hana-acciaioli) [reply src/components/Controls/Controls.js:48]: Thank you!!

**Pull request merged**